src/psyclone/configuration.py

```diff
@@ -65,6 +65,9 @@
 #          applied and only one version of the transformed kernel is created.
 VALID_KERNEL_NAMING_SCHEMES = ["multiple", "single"]
 
+# Specify a valid name for use in checking for variable names etc.
```

Comment:
Please could we encapsulate this functionality as a static method within the `Config` class?

Reply:
OK.